### PR TITLE
Formalize Sixpack Experiment

### PR DESCRIPTION
**Pull request by Vmack**
### What's this PR do?

This pull request moves social auth buttons to the top of the registration form permanently 

### How should this be reviewed?

👀

### Any background context you want to provide?

Users convert ~0.7% more when social signup buttons appear above registration form

### Relevant tickets

References [Pivotal #171413064](https://www.pivotaltracker.com/story/show/171413064).

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added appropriate feature/unit tests.
- [ ] If new attributes were added to users, there is a corresponding PR to surface these in Aurora.
- [ ] If new attributes were added to users, then the data team already knows about these changes.

### CODE REVIEW

File: app/Http/Controllers/Web/AuthController.php [hunk 1]
```diff
@@ -176,11 +176,10 @@ public function getLogout(Request $request)
      */
     public function getRegister()
     {
-        $positionSocialAuth = participate('social-auth-position', ['position_bottom', 'position_top']);
+        $positionSocialAuth = participate('social-auth-position', ['position_top']);
```
Comment (katiecrane) [inline app/Http/Controllers/Web/AuthController.php:179]: We can remove this whole line! `participate()` is what actually puts someone in an a/b test so we don't need it anymore 🔥

Comment (Vmack) [reply app/Http/Controllers/Web/AuthController.php:179]: ahah! ok gotcha!

File: resources/views/auth/register-beta.blade.php [hunk 2]
```diff
@@ -69,22 +69,6 @@
         </div>
     </form>
 
-    @if ($social_auth_position === 'position_bottom')
```
Comment (katiecrane) [inline resources/views/auth/register-beta.blade.php:72]: There's an `if` statement wrapped around the social share buttons up at the top - we can remove the conditional and just always display the buttons now!

**Comment by katiecrane** [comment]
This looks great so far! There are a few more remnants left in the codebase. When someone converts on an a/b test we need to track it so we have a few:
```js
convert('social-auth-position');
```
one in AuthController and then one in GoogleController and FacebookController too! So those can all be removed as well

**Pull request merged**